automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainMultiGroupApiSpec.scala

```diff
@@ -0,0 +1,418 @@
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        "when shared with one of the groups in the auth domain" - {
+          "can be seen and is accessible by group member who is a member of both auth domain groups" in {
+
+            val user = UserPool.chooseStudent
+            implicit val authToken: AuthToken = authTokenDefault
+
+            withGroup("AuthDomainOne", List(user.email)) { groupOne =>
+              withGroup("AuthDomainTwo", List(user.email)) { groupTwo =>
+
+                withCleanBillingProject(defaultUser) { projectName =>
+                  withWorkspace(projectName, "GroupsApiSpec_workspace", Set(groupOne, groupTwo), List(AclEntry(groupNameToEmail(groupOne), WorkspaceAccessLevel.Reader))) { workspaceName =>
+
+                    // user can see workspace and user can access workspace
+                    AuthDomainMatcher.checkVisibleAndAccessible(projectName, workspaceName, List(groupOne, groupTwo))(user.makeAuthToken())
+                  }
+                }
+              }
+            }
+          }
+
+          "can be seen but is not accessible by group member who is a member of only one auth domain group" in {
+            val user = UserPool.chooseStudent
+            implicit val authToken: AuthToken = authTokenDefault
+            withGroup("AuthDomainOne", List(user.email)) { groupOne =>
+              withGroup("AuthDomainTwo") { groupTwo =>
+                withCleanBillingProject(defaultUser) { projectName =>
+                  withWorkspace(projectName, "GroupsApiSpec_workspace", Set(groupOne, groupTwo), List(AclEntry(groupNameToEmail(groupOne), WorkspaceAccessLevel.Reader))) { workspaceName =>
+
+                    // user can see workspace but user cannot access workspace
+                    AuthDomainMatcher.checkVisibleNotAccessible(projectName, workspaceName)(user.makeAuthToken())
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        "when not shared with them" - {
+          "cannot be seen and is not accessible" in {
+            val user = UserPool.chooseStudent
+            implicit val authToken: AuthToken = authTokenDefault
+
+            withGroup("AuthDomainOne", List(user.email)) { groupOne =>
+              withGroup("AuthDomainTwo", List(user.email)) { groupTwo =>
+
+                withCleanBillingProject(defaultUser) { projectName =>
+                  withWorkspace(projectName, "GroupsApiSpec_workspace", Set(groupOne, groupTwo)) { workspaceName =>
+
+                    // user cannot see workspace and user cannot access workspace
+                    AuthDomainMatcher.checkNotVisibleNotAccessible(projectName, workspaceName)(user.makeAuthToken())
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        //TCGA controlled access workspaces use-case
+        "when the workspace is shared with the group" - {
```

Comment:
This test is identical to the test above "a workspace with multiple groups inside of it when the user is in all of the groups when shared with one of the groups in the auth domain can be seen and is accessible by group member who is a member of both auth domain groups" ([line 326](https://github.com/broadinstitute/rawls/blob/4868778fc9e8a5d9300f4a25596c6a1945ae286c/automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainMultiGroupApiSpec.scala#L326))

Reply:
yes, duplicated. I removed this test.